code/modules/client/preferences/entries/character/quirkoptions.dm

```diff
@@ -0,0 +1,186 @@
+/datum/preference/choiced/quirk_prosthetic_limb_location
+	category = PREFERENCE_CATEGORY_NON_CONTEXTUAL
+	preference_type = PREFERENCE_CHARACTER
+	db_key = "quirk_prosthetic_limb_location"
+	can_randomize = FALSE
```

Comment:
these can be randomized. this just determines if random body should randomize it, all of these, yes, that's fine